projects/js-packages/idc/components/idc-screen/screen-main.jsx

```diff
@@ -38,25 +42,56 @@ const ScreenMain = props => {
 			</h2>
 
 			<p>
-				{ createInterpolateElement(
-					customContent.mainBodyText ||
-						__(
-							'Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. ' +
-								'2 sites that are telling Jetpack they’re the same site. <safeModeLink>Learn more about safe mode.</safeModeLink>',
-							'jetpack'
-						),
-					{
-						safeModeLink: (
-							<a
-								href={ customContent.supportURL || getRedirectUrl( 'jetpack-support-safe-mode' ) }
-								rel="noopener noreferrer"
-								target="_blank"
-							/>
-						),
-						em: <em />,
-						strong: <strong />,
-					}
-				) }
+				{ ! isDevelopmentSite
+					? createInterpolateElement(
+							customContent.mainBodyText ||
+								__(
+									'Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. ' +
+										'2 sites that are telling Jetpack they’re the same site. <safeModeLink>Learn more about safe mode.</safeModeLink>',
```

Comment:
```suggestion
										'Two sites that are telling Jetpack they’re the same site. <safeModeLink>Learn more about safe mode.</safeModeLink>',
```
I know this is how it was before hand (with '2'), but technically it should be 'Two' at the beginning of the sentence (using the Chicago Style Guide), so it might be worth a quick update while we're looking at this code?